smart_card_connector_app/build/js_to_cxx_tests/Makefile

```diff
@@ -66,6 +66,7 @@ LIBS := \
 
 JS_SOURCES_PATHS := \
   $(SOURCE_DIR) \
+  !$(SOURCE_DIR)/**-unittest.js \
   $(SOURCE_DIR)/**-jstocxxtest.js \
```

Comment:
isn't this one already added by $(SOURCE_DIR) in line 68?

Reply:
You're right, it's not necessary. I guess I originally added it to make it more explicit, but let's remove it.